programs/mango-v4/src/state/mango_account.rs

```diff
@@ -257,9 +257,14 @@ pub struct MangoAccountFixed {
     pub buyback_fees_accrued_previous: u64,
     pub buyback_fees_expiry_timestamp: u64,
     pub next_token_conditional_swap_id: u64,
-    pub reserved: [u8; 200],
+    pub temporary_delegate: Pubkey,
+    pub temporary_delegate_expiry: u64,
+    pub reserved: [u8; 160],
```

Comment:
we're eating up bytes pretty quickly; luckily we could extend for more fixed storage if necessary at a later point